tests/epyccel/test_epyccel_python_annotations.py

```diff
@@ -73,7 +76,7 @@ def test_array_real_1d_scalar_add(language):
     f1(x1, a)
     f2(x2, a)
 
-    assert np.array_equal( x1, x2 )
+    assert np.allclose(x1, x2, rtol=RTOL, atol=ATOL)
```

Comment:
Adding small integers in floating-point representation should be exact, hence not affected by roundoff errors.